Strange CI failure:

```
Python 3.9.13
Error: venv/lib/python3.9/site-packages/iso4217/__init__.pyi:28: error: unexpected indent  [syntax]
Found 1 error in 1 file (errors prevented further checking)
Error: Process completed with exit code 2.
```

It looks like we have an indirect dependency on `iso4217`, which recently included code in its main stub file: https://github.com/dahlia/iso4217/blob/121b0ed983425f78f6c13a4f233ee870e44464fe/iso4217/__init__.pyi#L28.

--- a/homeassistant/components/zha/manifest.json
+++ b/homeassistant/components/zha/manifest.json
@@ -9,7 +9,7 @@
     "pyserial-asyncio==0.6",
     "zha-quirks==0.0.77",
     "zigpy-deconz==0.18.0",
-    "zigpy==0.47.2",
+    "zigpy==0.47.3",
     "zigpy-xbee==0.15.0",
     "zigpy-zigate==0.9.0",
     "zigpy-znp==0.8.1"

--- a/requirements_all.txt
+++ b/requirements_all.txt
@@ -2522,7 +2522,7 @@ zigpy-zigate==0.9.0
 zigpy-znp==0.8.1
 
 # homeassistant.components.zha
-zigpy==0.47.2
+zigpy==0.47.3
 
 # homeassistant.components.zoneminder
 zm-py==0.5.2

--- a/requirements_test_all.txt
+++ b/requirements_test_all.txt
@@ -1686,7 +1686,7 @@ zigpy-zigate==0.9.0
 zigpy-znp==0.8.1
 
 # homeassistant.components.zha
-zigpy==0.47.2
+zigpy==0.47.3
 
 # homeassistant.components.zwave_js
 zwave-js-server-python==0.39.0

